src/js/pouch.js

```diff
@@ -0,0 +1,62 @@
+// Utility functions to add pouch to an existing express instance
+"use strict";
+var fluid          = require('infusion');
+var namespace      = "gpii.pouch";
+var pouch          = fluid.registerNamespace(namespace);
+var PouchDB        = require("pouchdb");
+var memdown        = require('memdown');
+var expressPouchdb = require('express-pouchdb');
+
+pouch.addRoutesPrivate = function(that) {
```

Comment:
As before in gpii-express - break up and rename the functionality in this initialisation block